cylc/flow/task_state.py

```diff
@@ -184,6 +175,8 @@ class TaskState:
             External triggers as {trigger (str): satisfied (boolean), ...}.
         .is_held (bool):
             True if the task is "held" else False.
+        .is_queued (bool):
+            True if the task is queued else False.
```

Comment:
True if the task is "queued"? To follow the example above... (sorry to nit-pick!)

Reply:
(I'll remove the quotes on "held")